spec/javascripts/helpers/jasmine-jquery.js

```diff
@@ -0,0 +1,841 @@
+/*!
```

Comment:
Okay. A bit weird - maybe we should investigate integrating npm at some point (obviously not as part of this PR)